Framework/src/TaskRunner.cxx

```diff
@@ -485,4 +489,16 @@ int TaskRunner::publish(DataAllocator& outputs)
   return objectsPublished;
 }
 
+void TaskRunner::saveToFile()
+{
+  if (!mTaskConfig.saveToFile.empty()) {
+    ILOG(Debug, Support) << "Save data to file " << mTaskConfig.saveToFile << ENDM;
+    TFile f(mTaskConfig.saveToFile.c_str(), "RECREATE");
+    for (size_t i = 0; i < mObjectsManager->getNumberPublishedObjects(); i++) {
+      mObjectsManager->getMonitorObject(i)->getObject()->Write();
+    }
+    f.Close();
```

Comment:
But will that work if there is more than one MO? won't they overwrite each other?

Reply:
No, because ROOT saves the objects under a name (Key) which corresponds to the TObject name.

Reply:
thanks